src/one_dragon/base/operation/application/application_run_context.py

```
@@ -0,0 +1,318 @@
+        elif self._run_state == ApplicationRunContextStateEnum.PAUSE:
+            log.info("恢复运行")
+            self._run_state = ApplicationRunContextStateEnum.RUNNING
+            self._event_bus.dispatch_event(
+                ApplicationRunContextStateEventEnum.RESUME, self._run_state
+            )
+
+    def run_application(self, app_id: str, instance_idx: int, group_id: str) -> bool:
+        """
+        同步运行指定的应用。
+        
+        设置当前运行信息，创建应用实例并执行，最后清理运行状态。
```

Ruff (0.12.2)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)